dashboard/templates/base.html

```diff
@@ -47,7 +47,7 @@
                 {% if user.is_superuser and git_version %}
                 <td style="text-align: left">
                     Git version:
-                    <a href="{{ git_version.repo }}/commit/{{ git_version.commit }}">{{ git_version.commit_abbrev }}</a>
+                    <a href="{{ git_version.repo_abbrev }}/commit/{{ git_version.commit }}" target="_blank">{{ git_version.commit_abbrev }}</a>
```

Comment:
This looks good. 

While you're changing this can you make the {{git_version.branch}} on the next line 51 an anchor too?

So it would be in a link like
`<a href="{{ git_version.repo }}/tree/{{ git_version.branch }}" target="_blank">`

Reply:
Yes I can add this!